Library/Homebrew/utils/rubocop.rb

```diff
@@ -2,9 +2,7 @@
 # typed: false
 # frozen_string_literal: true
 
-require_relative "gems"
-Homebrew.setup_gem_environment!
-
+require_relative "../startup"
```

Comment:
Why does `rubocop` need this, out of interest?

Reply:
It requires the warning gem so we should be setting up the gem environment and the load path properly to use it.

This probably could have been `../load_path` - I'll double check.

Note that this file (unlike `rubocops.rb`) is only invoked with Rubocop runs via `brew`.

Reply:
> This probably could have been `../load_path` - I'll double check.

That'd be cool if so 👍🏻